### PR TITLE
Add CNI focussed node-density workloads

### PR DESCRIPTION
**Pull request by smalleni**
Signed-off-by: Sai Sindhur Malleni <smalleni@redhat.com>

### Description
e2e scripts to impleemnt the new kubelet-density workloads in https://github.com/cloud-bulldozer/benchmark-operator/pull/727

### CODE REVIEW

File: workloads/kube-burner/run_nodedensity-cni-networkpolicy_test_fromgit.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+METRICS_PROFILE=${METRICS_PROFILE:-metrics-profiles/metrics.yaml}
+NODE_COUNT=${NODE_COUNT:-4}
+PODS_PER_NODE=${PODS_PER_NODE:-250}
+export WORKLOAD=node-density-cni-networkpolicy
+
+. common.sh
+
+deploy_operator
+check_running_benchmarks
+label_nodes heavy
```
Comment (dry923) [inline workloads/kube-burner/run_nodedensity-cni-networkpolicy_test_fromgit.sh:13]: why are we labeling nodes w/ "heavy"? same for the other test.

File: workloads/kube-burner/run_nodedensity-cni_test_fromgit.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+METRICS_PROFILE=${METRICS_PROFILE:-metrics-profiles/metrics.yaml}
+NODE_COUNT=${NODE_COUNT:-4}
+PODS_PER_NODE=${PODS_PER_NODE:-250}
+export WORKLOAD=node-density-cni
+
+. common.sh
+
+deploy_operator
+check_running_benchmarks
+label_nodes heavy
```
Comment (dry923) [inline workloads/kube-burner/run_nodedensity-cni_test_fromgit.sh:13]: why are we labeling nodes w/ "heavy"? same for the other test.

File: workloads/kube-burner/run_nodedensity-cni-networkpolicy_test_fromgit.sh (hunk 1 above)
Comment (smalleni) [reply workloads/kube-burner/run_nodedensity-cni-networkpolicy_test_fromgit.sh:13]: ctrl C+v obviously, good catch

Comment (smalleni) [reply workloads/kube-burner/run_nodedensity-cni-networkpolicy_test_fromgit.sh:13]: well we'd still need a label - perhaps a different label but we assigns pods to nodes based on the label right? This is still a kubelet-density style of test.

Comment (smalleni) [reply workloads/kube-burner/run_nodedensity-cni-networkpolicy_test_fromgit.sh]: Just changed the label to cni.

File: workloads/kube-burner/run_nodedensity-cni_test_fromgit.sh (hunk 2 above)
Comment (smalleni) [reply workloads/kube-burner/run_nodedensity-cni_test_fromgit.sh]: Yeh we need *some* label, I just changed it to cni

**Comment by smalleni** [comment]
@dry923 @rsevilla87 Let me know if there's anything still holding this back please.

**Review by dry923 — APPROVED** [review]
LGTM!

**Comment by mohit-sheth** [comment]
need to update this PR with changes proposed here https://github.com/cloud-bulldozer/kube-burner/pull/165

**Review by mohit-sheth — APPROVED** [review]
LGTM

**Review by rsevilla87 — CHANGES_REQUESTED** [review]
Hey @smalleni, overall looks good, nice job. Can you update the docs?

**Review by mohit-sheth — APPROVED** [review]
LGTM

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Pull request merged**